docs/language/learn-ql/csharp/introduce-libraries-csharp.rst

```diff
@@ -1,18 +1,20 @@
-Introducing the CodeQL libraries for C#
-=======================================
+CodeQL libraries for C#
+=======================
 
-Overview
---------
+When you're analyzing a C# program in {{ site.data.variables.product.prodname_dotcom }}, you can make use of the large collection of classes in the CodeQL libraries for C#.
```

Comment:
A bit late to the review (apologies), but is the use of `{{ site.data.variables.product.prodname_dotcom }}` intentional here?

Reply:
I had my help-docs repo head on when I wrote that. 
That's not going to work here is it? I want to write GitHub obviously. What have you done elsewhere for this sort of thing?

Reply:
Ah, I see. A user could doing an analysis with the CodeQL CLI, CodeQL for VS Code, or the query console, so we probably don't want to mention a specific product here. Suggest changing to:

_When you're analyzing a C# program, you can make use of the large collection of classes in the CodeQL libraries for C#._

Reply:
👍 thanks
I'll push another PR to change this.

Reply:
Ouch - not sure how I managed to miss that! Thanks @jf205